Formula/c/catgirl.rb

```diff
@@ -0,0 +1,30 @@
+class Catgirl < Formula
+  desc "Terminal IRC client"
+  homepage "https://git.causal.agency/catgirl/about/"
+  url "https://git.causal.agency/catgirl/snapshot/catgirl-2.2a.tar.gz"
+  sha256 "c6d760aaee134e052586def7a9103543f7281fde6531fbcb41086470794297c2"
+  license "GPL-3.0-or-later"
+
+  depends_on "ctags" => :build
```

Comment:
```suggestion
  license "GPL-3.0-or-later"
  
  livecheck do
    url "https://git.causal.agency/catgirl"
    regex(/href=.*?catgirl[._-]v?(\d+(?:\.\d+)+[a-z]?)\.t/i)
  end

  depends_on "ctags" => :build
```
We need an explicit `livecheck` block as `brew livecheck` isn't able to fetch versions without one. The `Git` strategy doesn't seem to work so we can go with this, at least for now.